Ginger/GingerCore/Actions/ActScript.cs

```diff
@@ -305,5 +347,44 @@ public string GetCommandText(ActScript act)
                     return "Error - unknown command";
             }
         }
+
+        string CreateTempFile(string extenion)
+        {
+            string fileName = string.Empty;
+            try
+            {
+                if (!string.IsNullOrEmpty(SolutionFolder))
+                {
+                    fileName = Path.Combine(SolutionFolder, @"Documents\scripts\", Guid.NewGuid().ToString() + "." + extenion);
+                }
+                else
+                {
+                    fileName = Path.GetTempPath() + Guid.NewGuid().ToString() + "." + extenion;
+                }
+            }
+            catch (Exception ex)
+            {
+                //this to overcome speical IT settings which doesn't allow local personal folders
+                fileName = "" + Environment.GetFolderPath(Environment.SpecialFolder.InternetCache) + @"\" + Guid.NewGuid().ToString() + "." + extenion;
+            }
+            return fileName;
+        }
+
+        bool DeleteTempFile(string filePath)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change return type to 'void'; not a single caller uses the returned value.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5639590)